### PR TITLE
Create a process file application

### PR DESCRIPTION
**Pull request by blackinno**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,23 +4,46 @@
 
 # Installation
 
-```
-mesg-core service dev services/encryption
-```
-update .envrc ENCRYPT_ID
+## Install MESG Core
 
+Make sure that [MESG Core](https://github.com/mesg-foundation/core) is installed and running on your computer.
```
Comment (antho1404) [inline readme.md]:
```suggestion
Make sure that [MESG Engine](https://github.com/mesg-foundation/engine) is installed and running on your computer.
```

File: readme.md [hunk 2]
```diff
@@ -4,23 +4,46 @@
 
 # Installation
 
-```
-mesg-core service dev services/encryption
-```
-update .envrc ENCRYPT_ID
+## Install MESG Core
```
Comment (antho1404) [inline readme.md]:
```suggestion
## Install MESG Engine
```

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,5 @@
+PRIVATE_KEY=0x...
+SENDGRID_API_KEY=SG...
+PROVIDER_ENDPOINT=https://kovan.infura.io/v3/__PROJECT_ID__
+CONTRACT_ABI=`cat __PATH_OF_ABI_FILE`
```
Comment (antho1404) [inline .env.example]:
```suggestion
CONTRACT_ABI=`cat abi.json`
```

File: readme.md [hunk 4]
```diff
@@ -4,23 +4,46 @@
...
+
+## Deploy MESG process
+
+```bash
+mesg-cli process:dev application.yml \
+    --env PROVIDER_ENDPOINT=$PROVIDER_ENDPOINT \
+    --env PRIVATE_KEY=$PRIVATE_KEY \
+    --env SENDGRID_API_KEY=$SENDGRID_API_KEY \
+    --env CONTRACT_ABI=$CONTRACT_ABI
+    --env CONTRACT_ADDRESS=$CONTRACT_ADDRESS \
```
Comment (antho1404) [inline readme.md]: This command doesn't work

**Review by antho1404 — CHANGES_REQUESTED** [review]

File: application.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  - type: task
+    key: convert-string
+    instance:
+      src: https://github.com/mesg-foundation/service-js-function
+    taskKey: execute
+    inputs:
+      code: 'module.exports = value => value.status.toString()'
+      inputs:
+        status: {key: status}
+  - tasks: tasks
```
Comment (antho1404) [inline application.yml]:
```suggestion
  - type: task
```

File: application.yml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+      instance:
+        src: https://github.com/mesg-foundation/service-ethereum-contract
+      taskKey: execute
+        inputs:
+          method: editExecution
+          privateKey: $(env:PRIVATE_KEY)
```
Comment (antho1404) [inline application.yml]:
```suggestion
          privateKey: "$(env:PRIVATE_KEY)"
```

File: readme.md (hunk 4 above)
Comment (blackinno) [reply readme.md]: Fixed.

File: .env.example [hunk 7]
```diff
@@ -0,0 +1,5 @@
+PRIVATE_KEY=0x...
+SENDGRID_API_KEY=SG...
+PROVIDER_ENDPOINT=https://kovan.infura.io/v3/__PROJECT_ID__
+CONTRACT_ABI=`cat abi.json`
+CONTRACT_ADDRESS=0x..
```
Comment (antho1404) [inline .env.example]:
```suggestion
CONTRACT_ADDRESS=0x82d8496d668D2E1e298C7B061aA88498Fd068971
```

Comment (antho1404) [reply .env.example]: This is already hardcoded in the UI so we could have it on the env directly

**Review by antho1404 — APPROVED** [review]

**Pull request merged**